Ginger/Ginger/Activities/ActivityDetailsPage.xaml.cs

```diff
@@ -186,6 +200,16 @@ private void BindControls()
                 xHandlerTriggerOnStackPanel.Visibility = Visibility.Collapsed;
                 xHandlerPostExecutionActionStack.Visibility = Visibility.Collapsed;
             }
+
+            if (GetCurrentActivityPlatform() == ePlatformType.WebServices)
+            {
+                xConsumerStack.Visibility = Visibility.Visible;
+                xConsumerStackLogic();
+            }
+            else 
+            {
+                xConsumerStack.Visibility = Visibility.Collapsed;
```

Comment:
Make sure to clear binding which was added in xConsumerStackLogic function to clear incase target application of activity is changed to any platform other than wesbservices.

Reply:
added